packages/create-toolpad-app/src/index.ts

```diff
@@ -161,6 +161,323 @@ const scaffoldProject = async (absolutePath: string, installFlag: boolean): Prom
+                <NotificationsIcon />
+              </Badge>
+            </IconButton>
+          </Toolbar>
+        </AppBar>
+        <Drawer variant="permanent" anchor="left">
+          <Toolbar
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "flex-end",
+              px: [1],
+            }}
+          ></Toolbar>
+          <Divider />
+          <List component="nav">
+            <ListItemButton>
+              <ListItemIcon>
+                <HomeIcon />
+              </ListItemIcon>
+            </ListItemButton>
+            <ListItemButton>
+              <ListItemIcon>
+                <SettingsIcon />
+              </ListItemIcon>
+            </ListItemButton>
+          </List>
+        </Drawer>
+        <Box
+          component={"main"}
+          sx={{ flexGrow: 1, height: "100vh", overflow: "auto" }}
+        >
+          <Toolbar />
+          <Container maxWidth="lg">{children}</Container>
+        </Box>
+      </Box>
+    );
+  }
+`;
+  // Create the `layout.tsx` file inside the `(dashboard)` directory
+  await fs.writeFile(
+    path.join(absolutePath, 'app', '(dashboard)', 'layout.tsx'),
+    dashboardLayoutContent,
+  );
+
+  const rootLayoutContent = `
+  import { AppRouterCacheProvider } from '@mui/material-nextjs/v13-appRouter';
+  import { ThemeProvider } from '@mui/material/styles';
+  import theme from '../theme';
+  
+  
+    export default function RootLayout({ children }: Readonly<{ children: React.ReactNode }>) {      
+      return (
+        <html lang="en">
+          <body>
+            <AppRouterCacheProvider>
+            <ThemeProvider theme={theme}>
+                {children}
+              </ThemeProvider>
+            </AppRouterCacheProvider>
+          </body>
+        </html>
+      );
+    }
+    `;
+  // Write the content of the `layout.tsx` file
+  await fs.writeFile(path.join(absolutePath, 'app', 'layout.tsx'), rootLayoutContent);
+
+  const themeContent = `    
+  "use client";
+  import { Roboto } from "next/font/google";
+  import { createTheme } from "@mui/material/styles";
+  
+  const roboto = Roboto({
+    weight: ["300", "400", "500", "700"],
+    subsets: ["latin"],
+    display: "swap",
+  });
+  
+  const theme = createTheme({
+    typography: {
+      fontFamily: roboto.style.fontFamily,
+    },
+    components: {
+      MuiAppBar: {
+        styleOverrides: {
+          root: {
+            boxShadow: "none",
+          },
+        },
+      },
+      MuiList: {
+        styleOverrides: {
+          root: {
+            padding: 0,
+          },
+        },
+      },
+      MuiListItemIcon: {
+        styleOverrides: {
+          root: {
+            minWidth: "28px",
+          },
+        },
+      },
+    },
+  });
+  
+  export default theme;
+  `;
+
+  // Create the `theme.ts` file in the root directory
+  await fs.writeFile(path.join(absolutePath, 'theme.ts'), themeContent);
```

Comment:
So we're not using an `src` folder? I prefer to use one just to organize code but any option is fine, we just never discussed it.